### PR TITLE
[TRANSLATION] Add Indonesian Translation

### PR DESCRIPTION
**Pull request by Mas4hmad**
This is first commit of Indonesian Translation Language for Risoh Editor.

### CODE REVIEW

**Comment by katahiromz** [comment]
Can you add your name into text file `TRANSLATORS.txt`? Thanks.

**Comment by Mas4hmad** [comment]
> 
> 
> Can you add your name into text file `TRANSLATORS.txt`? Thanks.

Thanks for advice, i'll do it

File: src/lang/id_ID.rc [hunk 1]
```diff
@@ -0,0 +1,1583 @@
...
+            VALUE "FileDescription", "RisohEditor\0"
+            VALUE "FileVersion", "5.4.4\0"
+            VALUE "LegalCopyright", "Hak cipta (C) 2017-2019 Katayama Hirofumi MZ. Hak cipta dilindungi.\0"
+            VALUE "ProductName", "RisohEditor\0"
+            VALUE "ProductVersion", "5.4.4\0"
+        }
+    }
+    BLOCK "VarFileInfo"
+    {
+        VALUE "Translation", 0x0409, 0x04E4
```
Comment (katahiromz) [inline src/lang/id_ID.rc:1304]: 0x0409, 0x04E4
Can you confirm these values for Indonesian?

File: src/lang/id_ID.rc [hunk 2]
```diff
@@ -0,0 +1,1583 @@
...
+
+1 VERSIONINFO
+FILEVERSION     5, 4, 4, 0
+PRODUCTVERSION  5, 4, 4, 0
+FILEOS          0x40004
+FILETYPE        0x1
+{
+    BLOCK "StringFileInfo"
+    {
+        BLOCK "040904E4"
```
Comment (katahiromz) [inline src/lang/id_ID.rc:1292]: Also these values.

File: src/lang/id_ID.rc (hunk 1 above)
Comment (Mas4hmad) [reply src/lang/id_ID.rc:1304]: how to see that value?

Comment (katahiromz) [reply src/lang/id_ID.rc:1304]: https://docs.microsoft.com/id-id/windows/win32/menurc/varfileinfo-block

Comment (Mas4hmad) [reply src/lang/id_ID.rc:1304]: i don't see with 04e4 yet, what does meaning?
thanks

Comment (katahiromz) [reply src/lang/id_ID.rc:1304]: That is in hex display. Use calculator.

Comment (katahiromz) [reply src/lang/id_ID.rc:1304]: https://webcheatsheet.com/html/character_sets_list.php

File: src/lang/id_ID.rc (hunk 2 above)
Comment (Mas4hmad) [reply src/lang/id_ID.rc]: i found the charset. it's 4b0 in a web reference

File: src/lang/id_ID.rc (hunk 1 above)
Comment (Mas4hmad) [reply src/lang/id_ID.rc]: i get it. thanks

**Comment by katahiromz** [comment]
I will confirm the contents. Have a wait two days.

File: src/lang/id_ID.rc [hunk 3]
```diff
@@ -0,0 +1,1583 @@
...
+    cmb2, 0x0143, 0x0010, 0, 0x4243, 0x4D45, 0x495F, 0x534E, 0x5245, 0x4954, 0x4554, 0x004D, 
+    0
+}
+
+//////////////////////////////////////////////////////////////////////////////
+// RISOHTEMPLATE
+
+4 RISOHTEMPLATE "res/1033_RISOHTEMPLATE_4.bin"
+
+5 RISOHTEMPLATE "res/1033_RISOHTEMPLATE_5.bin"
```
Comment (katahiromz) [inline src/lang/id_ID.rc:1332]: Can  you add `(lang-id)_RISOHTEMPLATE_?.bin` files?

Comment (Mas4hmad) [reply src/lang/id_ID.rc:1332]: If i followed the code following before.
It must be
`4 RISOHTEMPLATE "res/(lang-id)_RISOHTEMPLATE_4.bin`

`5 RISOHTEMPLATE "res/(lang-id)_RISOHTEMPLATE_5.bin`

`6 RISOHTEMPLATE "res/(lang-id)_RISOHTEMPLATE_6.bin`?
I'll Check to other translation as comparison. Thanks

**Comment by katahiromz** [comment]
```txt
C:\projects\risoheditor\src\lang/id_ID.rc(1332): error RC2135: file not found: res/1057_RISOHTEMPLATE_5.bin [C:\projects\risoheditor\src\RisohEditorPortable.vcxproj]
```

**Comment by Mas4hmad** [comment]
> 
> 
> ```
> C:\projects\risoheditor\src\lang/id_ID.rc(1332): error RC2135: file not found: res/1057_RISOHTEMPLATE_5.bin [C:\projects\risoheditor\src\RisohEditorPortable.vcxproj]
> ```

thanks for noticing me and giving information what did must i do. i'll do

**Pull request merged**